.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE REQUEST] Short description of the feature request"
+labels: feature-request, status/grooming-needed
```

Comment:
shouln't it be `type/feature-request` to be consistent with `type/bug`?

Reply:
Sounds good -  It can be updated once merged, before that we can create just from scratch.